hw9/src/Homework.java

```diff
@@ -0,0 +1,48 @@
+import java.util.*;
+
+public class Homework {
+    public static int countOccurance(ArrayList<String> list, String elem) {
+        int result = 0;
+        for (int i = 0; i < list.size(); i++) {
+            if (list.get(i).equals(elem)) {
+                result++;
+            }
+        }
+        System.out.println("countOccurance(): В листе розмером: " + list.size() + " cлов " + elem + " встречается " + result + " раз");
+        return result;
+    }
+
+    public static ArrayList<String> toList(String arr[]) {
```

Comment:
The task is "convert an array into a list" - not `ArrayList` or any other specific list.
In other words, the method must return `List`.